sdk/lib/actions/input/builder/value.ts

```diff
@@ -73,28 +74,6 @@ function asRequiredParser<
   return parser.optional() as any
 }
 
-/**
```

Comment:
Why are you removing the documentation?

Reply:
@MattDHill

Reply:
Planning to individually document each type of value (toggle, text, color, etc).